app.rb

```diff
@@ -0,0 +1,21 @@
+require_relative './my_enumerable'
+
+class MyList
+  include MyEnumerable
+
+  def initialize(*list_items)
+    @list = list_items
+  end
+
+  def each(&block)
+    @list.each(&block)
+  end
+end
+
```

Comment:
- [ ] Please In #each method it should yield successive members of @list and uses the MyEnumerable module.
consider using yield keyword for this functionlity

Reply:
hello dear @MeqdamAlqudah thanks for your great review.
I just updated the code.

I'm fixing linters and resubmit for another review